### DISCUSSION TITLE
Report the Whatsapp numbers in the spam comments?

### DISCUSSION THREAD

**Discussion by kiatchun** [ThioJoe/YT-Spammer-Purge#199] [Ideas]
Is there a way that this script could report the numbers in the WhatsApp a certain number spam comments?

**Comment by Abelkrijgtalles** [comment]
Then you can use the [Whatsapp business api](https://www.whatsapp.com/business/api), or you can set up something like a bot with [Selenium](https://www.geeksforgeeks.org/whatsapp-using-python/)

**Reply by kiatchun** [reply]
Do you think, someone could implement a feature in this app to troll whatsapp scammers? (Then report them after they rage quit)

**Reply by Abelkrijgtalles** [reply]
> Do you think, someone could implement a feature in this app to troll whatsapp scammers? (Then report them after they rage quit)

I don't know, It could be possible, but that's not like really what this app for is

**Reply by Abelkrijgtalles** [reply]
> > Do you think, someone could implement a feature in this app to troll whatsapp scammers? (Then report them after they rage quit)
> 
> I don't know, It could be possible, but that's not like really what this app for is

You will then also need something like an ai

**Reply by KendallDoesCoding** [reply]
Well, @kiatchun even these whatsapp scammers setup bots in python to comment on EVERY single youtube video with their messege, using ascii characters and other stuff. A few scammers only, will themselves sit and comment on videos, so this trolling thing is a good concept but will be kinda useless according to me.

**Reply by kiatchun** [reply]
Then just report the numbers. I think it is harder to change numbers compared to channels. It will be more effective. (All the comments mentioning the reported number will not be useful anymore)

**Reply by KendallDoesCoding** [reply]
> Then just report the numbers. I think it is harder to change numbers compared to channels. It will be more effective. (All the comments mentioning the reported number will not be useful anymore)

Yes, this makes better sense, because one scammer will use like 5-10 bots to just share their one number. So if @ThioJoe makes it so that, you can report whatsapp scammers numbers, even if they use ascii characters that will be effective.

**Reply by Firecul** [reply]
> Do you think, someone could implement a feature in this app to troll whatsapp scammers? (Then report them after they rage quit)

That's a bad idea IMO. That takes this app from fighting scammers to becoming one. Not a good trade.

**Reply by kiatchun** [reply]
> > Do you think, someone could implement a feature in this app to troll whatsapp scammers? (Then report them after they rage quit)
> 
> That's a bad idea IMO. That takes this app from fighting scammers to becoming one. Not a good trade.

That is true.

**Reply by joaogelado** [reply]
dk too much, but I think twilio have a api

**Comment by Firecul** [comment]
That could be a neat idea, I imagine Whatsapp are more proactive when receiving reports of  spam etc, they couldn't be worse than YT at any rate.

**Reply by kiatchun** [reply]
I think the only problem is that you might need to expose your phone number to the scammers.
